tests/test_sklearn_scaler_converter.py

```diff
@@ -107,6 +109,33 @@ def test_float64_robust_scaler_floats(self):
         self.assertIsNotNone(torch_model)
         np.testing.assert_allclose(model.transform(data), torch_model.transform(data_tensor), rtol=1e-06, atol=1e-06)
 
+    # Tests with TVM backend
+    @unittest.skipIf(not tvm_installed(), reason="TVM test requires TVM installed")
+    def test_standard_scaler_floats_tvm(self):
+        # Generate a random 2D array with values in [0, 1000)
+        np.random.seed(0)
+        data = np.random.rand(100, 200) * 1000
+        data = np.array(data, dtype=np.float32)
+        data_tensor = torch.from_numpy(data)
+
+        model = StandardScaler(with_mean=False, with_std=False)
+        model.fit(data)
+        torch_model = hummingbird.ml.convert(model, "tvm", data, extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+        self.assertIsNotNone(torch_model)
+        np.testing.assert_allclose(model.transform(data), torch_model.transform(data_tensor), rtol=1e-06, atol=1e-06)
+
+        model = StandardScaler(with_mean=True, with_std=False)
+        model.fit(data)
+        torch_model = hummingbird.ml.convert(model, "tvm", data, extra_config={constants.TVM_MAX_FUSE_DEPTH: 30})
+        self.assertIsNotNone(torch_model)
+        np.testing.assert_allclose(model.transform(data), torch_model.transform(data_tensor), rtol=1e-04, atol=1e-04)
```

Comment:
did you mean this one to be `04` for `rtol/atol`?

Reply:
Nope. This one will actually fail if we increase the error. But this fails also in pytorch.I fixed it (for some reason we were using floats instead of doubles).